dags/xlml/solutionsteam_pax_nightly_supported.py

```diff
@@ -34,6 +34,32 @@
 ) as dag:
   log_dir_prefix = f"{gcs_bucket.XLML_OUTPUT_DIR}/pax/nightly"
 
+  # GPT-3 config with 1B params on c4 dataset with SPMD and Adam
+  c4spmd1b_pretraining_exp_path = "tasks.lm.params.c4.C4Spmd1BAdam4ReplicasLimitSteps"
+  pax_nightly_c4spmd1b_pretraining_v4_8 = pax_config.get_pax_lm_config(
```

Comment:
Do you want to add dependency relationship for `pax_nightly_c4spmd1b_pretraining_v4_8` and `pax_nightly_lmcloudspmdadam_v4_16`?

Reply:
These are two different tests, so I don't think there should be a dependency relationship.

Reply:
Nit: Probably you want to list those 2 tests in the `Test dependencies` section in the code (I know it will run on airflow without adding there)? otherwise, people may not miss those 2 tests during debugging.

Reply:
Will this code below work, or does there need to be a dependency defined (i.e, `pax_nightly_c4spmd1b_pretraining_v4_8 >> pax_nightly_lmcloudspmdadam_v4_16`)?

```
  # Test dependencies
  pax_nightly_c4spmd1b_pretraining_v4_8
  pax_nightly_lmcloudspmdadam_v4_16
  pax_nightly_lmspmd2b_v4_8 >> pax_nightly_lmspmd2b_ckpt_v4_8
  pax_nightly_lmtransformeradam_v4_8 >> pax_nightly_lmtransformeradam_v4_16
  pax_nightly_lmtransformeradam_v5p_4 >> pax_nightly_lmtransformeradam_v5p_16
  pax_nightly_lmtransformeradam_v5e_4 >> pax_nightly_lmtransformeradam_v5e_16
  ```

Reply:
The code should work, and you don't need to define the dependency.